packages/p2p/src/components/orders/popup.jsx

```diff
@@ -71,9 +71,10 @@ class Popup extends Component {
                                                             defaultChecked={
                                                                 values.need_confirmation
                                                             }
-                                                            label={localize(
-                                                                `I have received ${offer.currency} ${offer.fix_price}`,
-                                                            )}
+                                                            label={localize('I have received {{ offer }} {{ price }}', {
```

Comment:
we shouldn't have space in placeholders :(

Reply:
fixed 👍